src/gwt/org/bbop/apollo/gwt/client/dto/TrackInfo.java

```diff
@@ -84,6 +71,28 @@ public void setPayload(JSONObject payload) {
         this.payload = payload;
     }
 
+    public String getCategory() {
+        return category;
+    }
+
+    public void setCategory(String category) {
+        this.category = category;
+    }
+
+    public String getStandardCategory(){
+        if(category==null || category.trim().length()==0){
+            return "-----------";
+//            return "        ";
+        }
+        else{
+            String categoryString = "";
+            for(String c : category.split("\\/")){
+                categoryString+=c +"/";
```

Comment:
Tried doing `"category": "A/B/C"` but I end up with a category called 'A/B/C' on the Track Panel. I assume this is expected?